test/over_react_redux/hooks/use_selector_test.dart

```diff
@@ -1,183 +0,0 @@
-                (ReduxProvider()
-                  ..store = bigCounterStore
-                  ..context = bigCounterContext
-                )(
-                  CustomContextCounterFn()(),
-                  // Use a sibling connected component for dispatching actions in these tests
-                  // that shouldn't rely on `useDispatch` to ensure the subscription to context is wired up correctly
-                  (ConnectedBigCounter()..addTestId('big-sibling-counter'))(),
-                ),
-                // Use a sibling connected component for dispatching actions in these tests
-                // that shouldn't rely on `useDispatch` to ensure the subscription to context is wired up correctly
-                (ConnectedCounter()..addTestId('sibling-counter'))(),
-              ), attachedToDocument: true);
-
-          expectBigCountValue(jacket, 9);
-          await clickSiblingConnectedBigCountIncrementButton(jacket);
-          expectBigCountValue(jacket, 109);
-
-          expectCountValue(jacket, 0);
-          await clickSiblingConnectedIncrementButton(jacket);
-          expectCountValue(jacket, 1);
-        });
-
-        test('unless a custom equalityFn returns true', () async {
-          jacket = mount(
-              (ReduxProvider()..store = counterStore)(
-                (ReduxProvider()
-                  ..store = bigCounterStore
-                  ..context = bigCounterContext
-                )(
-                  (CustomContextCounterFn()..bigCountEqualityFn = (nextBigCount, prevBigCount) {
-                    // Use 209 so that the equalityFn returns false once, and then returns true.
-                    return nextBigCount == 209;
-                  })(),
-                  // Use a sibling connected component for dispatching actions in these tests
-                  // that shouldn't rely on `useDispatch` to ensure the subscription to context is wired up correctly
-                  (ConnectedBigCounter()..addTestId('big-sibling-counter'))(),
-                ),
-                // Use a sibling connected component for dispatching actions in these tests
-                // that shouldn't rely on `useDispatch` to ensure the subscription to context is wired up correctly
-                (ConnectedCounter()..addTestId('sibling-counter'))(),
-              ), attachedToDocument: true);
-
-          expectBigCountValue(jacket, 9);
-          await clickSiblingConnectedBigCountIncrementButton(jacket);
-          expectBigCountValue(jacket, 109, reason: 'Component should update when equalityFn returns false');
-          await clickSiblingConnectedBigCountIncrementButton(jacket);
-          expectBigCountValue(jacket, 109, reason: 'Component should not update when equalityFn returns true');
-
-          expectCountValue(jacket, 0);
-          await clickSiblingConnectedIncrementButton(jacket);
-          expectCountValue(jacket, 1);
-        });
-      });
-    });
-  });
-}
```

Comment:
These tests have moved to store_bindings_tests.dart. I started by refactoring them to use custom components so they could be run with multiple types of selected values, and they eventually morphed into tests that could also be shared with connect, thus the new file name.